tests/Jellyfin.Common.Tests/Providers/ProviderIdParserTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using MediaBrowser.Common.Providers;
+using Xunit;
+
+namespace Jellyfin.Common.Tests.Providers
+{
+    public class ProviderIdParserTests
+    {
+        [Theory]
+        [InlineData("tt123456", false, null)]
```

Comment:
Split the successful up from the failing
And follow the testing naming convention